pkgs/development/python-modules/default.nix

```diff
@@ -0,0 +1,41 @@
+{ pkgs
+, stdenv
+, interpreter
+#, interpreterConfig ? (self: super: {})
+, overrides ? (self: super: {})
+}:
+
+
+let
+
+  packages = let
+
+    inherit (stdenv.lib) fix' extends fold;
+
+    pythonPackages = self:
+      let
+        python = interpreter;
+      in import ./support.nix { inherit pkgs python; } self;
```

Comment:
I think the following would be a better way to write this:

```nix
let
...
pythonPackages = import ./support.nix {
  inherit pkgs; 
  python = interpreter;
};
```

Avoids the let, so it is more clear that the `python` is only used to pass to `support.nix`.